twilio/rest/api/v2010/__init__.py

```diff
@@ -1,231 +1,52 @@
-# coding=utf-8
-r"""
-This code was generated by
-\ / _    _  _|   _  _
- | (_)\/(_)(_|\/| |(/_  v1.0.0
-      /       /
+"""
+    This code was generated by
+    ___ _ _ _ _ _    _ ____    ____ ____ _    ____ ____ _  _ ____ ____ ____ ___ __   __
+    |  | | | | |    | |  | __ |  | |__| | __ | __ |___ |\ | |___ |__/ |__|  | |  | |__/
+    |  |_|_| | |___ | |__|    |__| |  | |    |__] |___ | \| |___ |  \ |  |  | |__| |  \
+
+    Twilio - Api
+    This is the public Twilio REST API.
+
+    NOTE: This class is auto generated by OpenAPI Generator.
+    https://openapi-generator.tech
+    Do not edit the class manually.
 """
 
 from twilio.base.version import Version
-from twilio.rest.api.v2010.account import AccountContext
+from twilio.base.domain import Domain
 from twilio.rest.api.v2010.account import AccountList
+from twilio.rest.api.v2010.account import AccountContext
 
 
 class V2010(Version):
 
-    def __init__(self, domain):
+    def __init__(self, domain: Domain):
         """
-        Initialize the V2010 version of Api
+        Initialize the V2010 version of api
 
-        :returns: V2010 version of Api
-        :rtype: twilio.rest.api.v2010.V2010.V2010
+        :param domain: The Twilio.api domain
         """
-        super(V2010, self).__init__(domain)
+        super().__init__(domain)
         self.version = '2010-04-01'
         self._accounts = None
         self._account = None
-
+        
     @property
-    def accounts(self):
-        """
-        :rtype: twilio.rest.api.v2010.account.AccountList
-        """
+    def accounts(self) -> AccountList:
         if self._accounts is None:
             self._accounts = AccountList(self)
         return self._accounts
 
     @property
-    def account(self):
-        """
-        :returns: Account provided as the authenticating account
-        :rtype: AccountContext
-        """
+    def account(self) -> AccountContext:
         if self._account is None:
             self._account = AccountContext(self, self.domain.twilio.account_sid)
         return self._account
 
-    @account.setter
-    def account(self, value):
```

Comment:
Is there a separate ticket for adding these back?

Reply:
Not that I'm aware of. This seems to be the only instance of this. Do we need it? I can create a ticket if so

Reply:
Yes.